ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -109,6 +109,20 @@ tests:
         VIP_DHCP_ALLOCATION=no
         PROXY=yes
     workflow: baremetalds-assisted-install
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-assisted-single-node
+  steps:
+    cluster_profile: packet
+    dependencies:
+      ASSISTED_SERVICE_IMAGE: pipeline:assisted-service
+    env:
+      ASSISTED_CONFIG: |
+        NUM_MASTERS=1
+        NUM_WORKERS=0
```

Comment:
you can remove it, ``NUM_MASTERS=1`` is implicitly inferring ``NUM_WORKERS=0```
Unless you prefer expressing it explicitly, then you can leave it as is

Reply:
This is just copy-paste from the infra config. Doesn't hurt to be explicit, I guess